lib/csetgrp.gi

```diff
@@ -1045,7 +1113,12 @@ local c, flip, maxidx, refineChainActionLimit, cano, tryfct, p, r, t,
   stabs:=[b];
   dcs:=[];
 
+  # Do we want to keep for smaller group (as cheaper fuse is possible
```

Comment:
I don't understand this sentence, "Do we want to keep for smaller group" -- keep what?